db/migrate/20190516214846_create_reddit_channel_details.rb

```diff
@@ -0,0 +1,14 @@
+class CreateRedditChannelDetails < ActiveRecord::Migration[5.2]
+  def change
+    create_table :reddit_channel_details, id: :uuid, default: -> { "uuid_generate_v4()"}, force: :cascade do |t|
+      t.string   "reddit_channel_id"
```

Comment:
I think we should add an index to this so that we don't run into any bottlenecks

Reply:
Good idea c414343